notebooks/README.md

```diff
@@ -0,0 +1,39 @@
+# Trtion FIL backend with XGBoost
+
+This notebook is a reference for deploying a forest model trained using XGBoost library in Triton Inference Server with Forest Inference Library (FIL) backend. The notebook explains how one can deploy XGBoost model in Triton, check deployment status and send inference requests, set concurrent model execution and dynamic batching and find the best deployment configuration using Model Analyzer.
+
+## Requirements
+* Nvidia GPU (T4 or V100 or A100)
```

Comment:
Let's make it clear that these are not the *only* GPUs that work for this, just the recommended ones.

Reply:
It would be best to specify the GPU architecture (i.e. Pascal+ or Volta+) as well as recommended GPUs